content/docs/quickstart/node.md

```diff
@@ -2,18 +2,29 @@
 title: Run a Node.js app
 ---
 
-1. Add postgres client to your project. In this example we used [postgres.js](https://www.npmjs.com/package/postgres), but feel free to choose another one
-2. Store your Neon credentials somewhere, for example in the `.env` file.
-
-   ```shell
-   NEON_HOST=...
-   NEON_DB=...
-   NEON_USER=...
-   NEON_PASS=...
-   NEON_PORT=...
-   ```
-
-3. To connect to the database using postgres client and your Neon credentials, add the following code to the `pages/api/hello_worlds.js`
+### Introduction
+
+In this section, you will learn how to add a Postgres client to your Project and connect to your database.
+
+## Step 1 — Add Postgres client to your Neon Project
+
+First, add a Postgres client to your Project. In this example, we used [postgres.js](https://www.npmjs.com/package/postgres), but feel free to choose another one.
+
+## Step 2 — Store Neon Credentials
+
+Next, store your Neon credentials somewhere, for example, in the `.env` file.
+
+```shell
+NEON_HOST=...
+NEON_DB=...
+NEON_USER=...
+NEON_PASS=...
+NEON_PORT=...
```

Comment:
it's probably better to be consistent with naming in our code examples modal. 
![code_examples](https://user-images.githubusercontent.com/3032829/173001821-e2400b2a-47d5-4bb9-a508-a3176100d7ca.png)

Reply:
Here we actually take them from env and manually put params into Postgres driver bellow.

But yeah, we can use standard libpq env variables https://www.postgresql.org/docs/14/libpq-envars.html, so that driver will pick them up automatically.

Not sure what is better. One way is more explicit, and one is more concise